packages/scaffold-ui/src/modal/w3m-router/index.ts

```diff
@@ -75,6 +75,22 @@ export class W3mRouter extends LitElement {
         return html`<w3m-connecting-external-view></w3m-connecting-external-view>`
       case 'ConnectingSiwe':
         return html`<w3m-connecting-siwe-view></w3m-connecting-siwe-view>`
+      case 'Networks':
+        return html`<w3m-networks-view></w3m-networks-view>`
+      case 'SwitchNetwork':
+        return html`<w3m-network-switch-view></w3m-network-switch-view>`
+      case 'Profile':
+        return html`<w3m-profile-view></w3m-profile-view>`
+      case 'SelectAddresses':
+        return html`<w3m-select-addresses-view></w3m-select-addresses-view>`
+      case 'SwitchAddress':
+        return html`<w3m-switch-address-view></w3m-switch-address-view>`
+      case 'WhatIsAWallet':
+        return html`<w3m-what-is-a-wallet-view></w3m-what-is-a-wallet-view>`
+      case 'WhatIsANetwork':
+        return html`<w3m-what-is-a-network-view></w3m-what-is-a-network-view>`
+      case 'GetWallet':
+        return html`<w3m-get-wallet-view></w3m-get-wallet-view>`
```

Comment:
Previously it was looking sorted alphabetically, why these are moved different rows?

Reply:
merge conflict, fixed